usaspending_api/search/tests/integration/test_spending_by_transaction.py

```diff
@@ -122,9 +123,9 @@ def test_all_fields_returned(client, transaction_data, elasticsearch_transaction
 
 
 @pytest.mark.django_db
-def test_subset_of_fields_returned(client, transaction_data, elasticsearch_transaction_index):
-
-    elasticsearch_transaction_index.update_index()
+def test_subset_of_fields_returned(client, monkeypatch, transaction_data, elasticsearch_transaction_index):
+    logging_statements = []
+    setup_elasticsearch_test(monkeypatch, elasticsearch_transaction_index, logging_statements)
```

Comment:
Nothing to block on, but using this method here is a bit overkill since it is intended to also handle the log check with experimental elasticsearch functionality within spending_by_category based endpoints.